edx_repo_tools/codemods/django3/github_actions_modernizer_django.py

```diff
@@ -35,13 +35,37 @@ def _update_django_matrix_items(self, job_name, job):
         matrices = job.get('strategy').get('matrix').items()
         for matrix_item_key, matrix_item in matrices:
             self._update_matrix_items(job_name, matrix_item_key, matrix_item)
+    
+    def _update_codecov_check(self, job_name, step): 
+        step_elements =  deepcopy(step)
+        if not 'uses' in step_elements:
+            return
+        if not (step_elements['uses']) in ['codecov/codecov-action@v1', 'codecov/codecov-action@v2']:
+            return
+        if not 'if' in step_elements:
+            return
+        step_index = self.elements['jobs'][job_name]['steps'].index(step)
+        django_32_string = step_elements['if'].replace('django22','django32')
+        self.elements['jobs'][job_name]['steps'][step_index]['if'] = django_32_string
 
+    def _update_job_steps(self, job_name, job):
+        steps = job.get('steps')
+        if not steps:
+            return
+        for step in steps:
+            self._update_codecov_check(job_name, step)
+
+    def _update_job(self):
```

Comment:
the method name should tell what's going on, like the one named `update_job_matrices`

Reply:
These names were meant to be generic since we can update more steps in future and hence this is a hierarchy from generic to specific functionality.